packages/vm/tests/util.ts

```diff
@@ -188,38 +195,40 @@ export function verifyAccountPostConditions(
       ] = acctData.storage[key]
     }
 
-    if (storageKeys.length > 0) {
-      state.root = account.stateRoot
-      const rs = state.createReadStream()
-      rs.on('data', function (data: any) {
-        let key = data.key.toString('hex')
-        const val = '0x' + rlp.decode(data.value).toString('hex')
-
-        if (key === '0x') {
-          key = '0x00'
-          acctData.storage['0x00'] = acctData.storage['0x00']
-            ? acctData.storage['0x00']
-            : acctData.storage['0x']
-          delete acctData.storage['0x']
-        }
+    state.root = account.stateRoot
+    const rs = state.createReadStream()
+    rs.on('data', function (data: any) {
+      let key = data.key.toString('hex')
+      const val = '0x' + rlp.decode(data.value).toString('hex')
+
+      if (key === '0x') {
+        key = '0x00'
+        acctData.storage['0x00'] = acctData.storage['0x00']
+          ? acctData.storage['0x00']
+          : acctData.storage['0x']
+        delete acctData.storage['0x']
+      }
 
-        t.equal(val, hashedStorage[key], 'correct storage value')
-        delete hashedStorage[key]
-      })
+      if (val !== hashedStorage[key]) {
+        t.comment(
+          `Expected storage key 0x${data.key.toString('hex')} at address ${address} to have value ${
```

Comment:
Have to add address here since it is not clear when these promises are fulfilled.